bait-files/src/main/webapp/WEB-INF/register.jsp

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
This file is not needed for this PR right?

Reply:
Removed